packages/flutter/lib/src/widgets/shortcuts.dart

```diff
@@ -1068,7 +1068,7 @@ class _ShortcutsState extends State<Shortcuts> {
   @override
   Widget build(BuildContext context) {
     return Focus(
-      debugLabel: '$Shortcuts',
+      debugLabel: '$Shortcuts${debugLabel == null ? '' : ' -- $debugLabel'}',
```

Comment:
```suggestion
      debugLabel: '$Shortcuts${debugLabel?.isEmpty ?? true ? '' : ' -- $debugLabel'}',
```

To also print nothing when the debug label is the empty string.